spec/routing/hq/dashboard_routing_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe 'Hq::Dashboard routing', type: :routing, pending: true do
+
+  specify { expect(get: '/hq/dashboard'). to be_routable }
+
+end
+
+RSpec.describe "Hq::Dashboard redirecting", type: :request, pending: true do
+  before :each do
+    sign_in instance_double(AdminUser)
+  end
+
+  it "root redirects to dashboard" do
+    get '/hq'
+    expect(response).to redirect_to hq_dashboard_path
+  end
+end
```

Comment:
What I meant is instead of a comment in the routes file which can be forgotten or overlooked, you could put something like this in, and it will force you to edit code and read the comment once the route exists.
``` ruby
  RSpec.describe 'Hq::Dashboard redirecting', type: :routing do
    specify 'disposable spec', pending: 'be sure to route the default "/hq" route here once available' do
      expect(get: '/hq/dashboard').to be_routable
    end
  end
```